config/install.yaml

```diff
@@ -455,7 +455,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: quay.io/numaproj/numaplane-controller:latest
+        image: quay.io/numaproj/numaplane-controller:v0.1.0-beta.3
```

Comment:
This tag change has to be merged?

Reply:
sorry, that's a mistake when I was testing